### DISCUSSION TITLE
Generated tables overflow parent container.

### DISCUSSION THREAD

**Discussion by mindaugasvcs** [bootstrap-email/bootstrap-email#150] [Help]
Hi,
It seems this tool still needs a lot of effort to make it usable.
Padding sometimes does not work or work as margin instead.
Generated tables overflow parent container.
How do I fix it?

**Comment by stuyam** [comment]
Feel free to post an example of code that you are struggling with and I would be happy to help 👍

**Reply by mindaugasvcs** [reply]
Moved over to mjml.io, problems solved.

**Reply by stuyam** [reply]
That's great, glad you found something that works for you 🎉
